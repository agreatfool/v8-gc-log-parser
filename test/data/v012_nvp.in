[87623]    21738 ms: pause=1.2 mutator=69.5 gc=s external=0.0 mark=0.0 sweep=0.00 sweepns=0.00 sweepos=0.00 sweepcode=0.00 sweepcell=0.00 sweepmap=0.00 evacuate=0.0 new_new=0.0 root_new=0.0 old_new=0.0 compaction_ptrs=0.0 intracompaction_ptrs=0.0 misc_compaction=0.0 weakcollection_process=0.0 weakcollection_clear=0.0 weakcollection_abort=0.0 total_size_before=410249136 total_size_after=396809216 holes_size_before=12582792 holes_size_after=12614440 allocated=14873496 promoted=1428088 semi_space_copied=1435056 nodes_died_in_new=29 nodes_copied_in_new=2 nodes_promoted=0 promotion_rate=8.7% semi_space_copy_rate=8.7% steps_count=0 steps_took=0.0 
[87623]    21795 ms: pause=4.8 mutator=52.5 gc=ms external=0.0 mark=0.6 sweep=3.71 sweepns=2.72 sweepos=0.33 sweepcode=0.28 sweepcell=0.01 sweepmap=0.07 evacuate=0.0 new_new=0.1 root_new=0.0 old_new=0.0 compaction_ptrs=0.0 intracompaction_ptrs=0.0 misc_compaction=0.1 weakcollection_process=0.0 weakcollection_clear=0.0 weakcollection_abort=0.0 total_size_before=406663248 total_size_after=391420328 holes_size_before=12614440 holes_size_after=13991208 allocated=9854032 promoted=1409952 semi_space_copied=1626888 nodes_died_in_new=16 nodes_copied_in_new=2 nodes_promoted=1 promotion_rate=12.7% semi_space_copy_rate=14.6% steps_count=17 steps_took=9.0 longest_step=2.1 incremental_marking_throughput=848273
