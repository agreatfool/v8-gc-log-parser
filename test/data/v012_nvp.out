[
  {
    "pid": 87623,
    "version": "v0.12.x",
    "time_since_init": 21738,
    "data": {
      "pause": 1.2,
      "mutator": 69.5,
      "gc": "s",
      "external": 0,
      "mark": 0,
      "sweep": 0,
      "sweepns": 0,
      "sweepos": 0,
      "sweepcode": 0,
      "sweepcell": 0,
      "sweepmap": 0,
      "evacuate": 0,
      "new_new": 0,
      "root_new": 0,
      "old_new": 0,
      "compaction_ptrs": 0,
      "intracompaction_ptrs": 0,
      "misc_compaction": 0,
      "weakcollection_process": 0,
      "weakcollection_clear": 0,
      "weakcollection_abort": 0,
      "total_size_before": 410249136,
      "total_size_after": 396809216,
      "holes_size_before": 12582792,
      "holes_size_after": 12614440,
      "allocated": 14873496,
      "promoted": 1428088,
      "semi_space_copied": 1435056,
      "nodes_died_in_new": 29,
      "nodes_copied_in_new": 2,
      "nodes_promoted": 0,
      "promotion_rate": 8.7,
      "semi_space_copy_rate": 8.7,
      "steps_count": 0,
      "steps_took": 0
    },
    "type": "nvp"
  },
  {
    "pid": 87623,
    "version": "v0.12.x",
    "time_since_init": 21795,
    "data": {
      "pause": 4.8,
      "mutator": 52.5,
      "gc": "ms",
      "external": 0,
      "mark": 0.6,
      "sweep": 3.71,
      "sweepns": 2.72,
      "sweepos": 0.33,
      "sweepcode": 0.28,
      "sweepcell": 0.01,
      "sweepmap": 0.07,
      "evacuate": 0,
      "new_new": 0.1,
      "root_new": 0,
      "old_new": 0,
      "compaction_ptrs": 0,
      "intracompaction_ptrs": 0,
      "misc_compaction": 0.1,
      "weakcollection_process": 0,
      "weakcollection_clear": 0,
      "weakcollection_abort": 0,
      "total_size_before": 406663248,
      "total_size_after": 391420328,
      "holes_size_before": 12614440,
      "holes_size_after": 13991208,
      "allocated": 9854032,
      "promoted": 1409952,
      "semi_space_copied": 1626888,
      "nodes_died_in_new": 16,
      "nodes_copied_in_new": 2,
      "nodes_promoted": 1,
      "promotion_rate": 12.7,
      "semi_space_copy_rate": 14.6,
      "steps_count": 17,
      "steps_took": 9,
      "longest_step": 2.1,
      "incremental_marking_throughput": 848273
    },
    "type": "nvp"
  }
]